package.json

```diff
@@ -58,9 +58,9 @@
     "release": "changeset publish",
     "docs": "yarn workspace @chakra-ui/docs",
     "docs:deploy": "yarn build && yarn docs:build-ci && cp -r website/public .",
-    "docs:build-ci": "yarn members:gen && yarn docs:build",
-    "docs:build": "yarn avatars:gen && yarn search-meta:gen && yarn docs build",
-    "docs:dev": "yarn avatars:gen && yarn docs dev",
+    "docs:build-ci": "yarn docs:build",
```

Comment:
Could you remove the `docs:build-ci` script? We can just use `yarn docs:build` in `docs:deploy` now.

Reply:
@with-heart now removed